components/PerangkatDesa/index.vue

```diff
@@ -35,7 +35,7 @@ export default {
     return {
       search: '',
       headerTableUserDesa,
-      statusPartner,
+      userStatus,
```

Comment:
is this `userData` be used in the future?

Reply:
it will use in the future after the endpoint from the backend team will ready. So, we just provide the data and create the template, first.

Reply:
i see, thanks 👍🏻